### PR TITLE
Microsoft look back

### PR DESCRIPTION
**Pull request by eyalpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XDRSUP-12076)

## Description
Look back mechanism was added to fetch-incidents

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@eyalpalo @dantavori what is the status here?

**Comment by eyalpalo** [comment]
@ShahafBenYakir Waiting for the CS to confirm that it is working properly on the customer side.

**Review by yaakovpraisler — APPROVED** [review]
Nice!

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md [hunk 1]
```diff
@@ -24,8 +24,18 @@ For more information about which permissions are required for the token owner in
     | First fetch time | First fetch timestamp \(&amp;lt;number&amp;gt; &amp;lt;time unit&amp;gt;, e.g., 12 hours, 7 days\) | False |
     | Incident resolution status |  | False |
     | Custom Filter | A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the other filters from the integration instance configuration. An example of a Custom Filter is: \{"severity":\{"eq":2\}\}. Note that for filtering by "entity.policy", you should use the ID of the policy. For example, for retrieving the policy: \{"policyType": "ANOMALY_DETECTION", "id": "1234", "label": "Impossible travel", "type": "policyRule"\}" please query on \{"entity.policy":\{"eq":1234\}\}. For more information about filter syntax, refer to https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters. | False |
+    | lookback | Advanced: Minutes to look back when fetching. | False
```
Comment (dantavori) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md:27]:
```suggestion
    | Advanced: Minutes to look back when fetching | Use this parameter to determine how long backward to look in the search for incidents to ensure collecting all incidents. | False
```
please update the yml too.
Also, what is the value we recommend?

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md [hunk 2]
```diff
@@ -24,8 +24,18 @@ For more information about which permissions are required for the token owner in
...
+
+
+5. Click **Test** to validate the URLs, token, and connection.
+
+
+
+## look-back parameter note
+In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing 
+the look back to a number that is greater than the previous value, then in the initial incident fetching there will be incidents duplications.
```
Comment (dantavori) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md:36]:
```suggestion
the look back to a number that is greater than the previous value, then in the next fetch there might be incidents duplications.
```

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md [hunk 3]
```diff
@@ -24,8 +24,18 @@ For more information about which permissions are required for the token owner in
...
+If the integration was already set with look back > 0, and the look-back is not being increased at any point of time, then those incident duplications would not occur.
```
Comment (dantavori) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md:37]:
```suggestion
```

File: Packs/MicrosoftCloudAppSecurity/ReleaseNotes/2_0_1.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft Cloud App Security
+- Added support for the *lookback* parameter in ***fetch-incidents*** to allow fetching older incidents.
```
Comment (dantavori) [inline Packs/MicrosoftCloudAppSecurity/ReleaseNotes/2_0_1.md:4]:
```suggestion
- Added support for the *lookback* parameter in the ***fetch-incidents*** command to ensure fetching all incidents.
```

**Pull request merged**